### DISCUSSION TITLE
2022/12/12/tuckermanravinehike

### DISCUSSION THREAD

**Discussion by giscus** [horenbergerb/horenbergerb.github.io#18] [Announcements]
# 2022/12/12/tuckermanravinehike

Context Context

https://horenbergerb.github.io/2022/12/12/tuckermanravinehike.html

**Comment by kbonney** [comment]
Nice hike, Beau! I hear that crampons can collect wet snow in clumps that make them less effective at gripping the ice. Something to consider if you make the investment.
